app/bundles/ReportBundle/Controller/ReportController.php

```diff
@@ -470,7 +472,18 @@ public function newAction($entity = null)
         $page    = $session->get('mautic.report.page', 1);
 
         $action = $this->generateUrl('mautic_report_action', ['objectAction' => 'new']);
-        $form   = $model->createForm($entity, $this->get('form.factory'), $action);
+        $form   = $model->createForm($entity, $this->get('form.factory'), $action, [
+            'constraints' => [
```

Comment:
@davevurby Let's move this to the defaults of the ReportType class https://github.com/mautic/mautic/blob/194654cd10b1ebf6e61d3b465bbf9ca47f90d5e8/app/bundles/ReportBundle/Form/Type/ReportType.php#L440